src/tree/tenants/registerTenantTree.ts

```diff
@@ -56,8 +56,10 @@ async function updateTenantsSetting(_context: IActionContext, tenants: vscode.Tr
 
     for (const [tenantTreeItem, state] of tenants.items) {
         if (state === vscode.TreeItemCheckboxState.Unchecked) {
+            _context.telemetry.properties.uncheckedTenant = 'true';
```

Comment:
We use this now so remove the underscore.

```suggestion
            context.telemetry.properties.uncheckedTenant = 'true';
```